platform/darwin/src/MGLTypes.h

```diff
@@ -95,10 +95,15 @@ typedef struct __attribute__((objc_boxable)) MGLTransition {
      The amount of time in seconds to wait before beginning the animation.
      */
     NSTimeInterval delay;
+    
+    /**
+     A boolean indicating whether label placement transitions are enabled.
+     */
+    BOOL enablePlacementTransitions;
```

Comment:
This changes the size of an instance of `MGLTransition`, which means a transition encoded using `+[NSValue valueWithMGLTransition:]` may not correctly decode using `NSValue.mglTransitionValue`.